app/presenters/updatable.rb

```diff
@@ -20,13 +25,21 @@ def history
     end
   end
 
-private
+  # The direction of change history isn’t guaranteed
+  # https://github.com/alphagov/govuk-content-schemas/issues/545
+  def reverse_chronological_change_history
+    change_history.sort_by { |item| DateTime.parse(item[:timestamp]) }.reverse
+  end
 
   def any_updates?
-    if (first_public_at = content_item["details"]["first_public_at"]).present?
+    if first_public_at.present?
       DateTime.parse(content_item["public_updated_at"]) != DateTime.parse(first_public_at)
     else
       false
     end
   end
+
+  def first_public_at
+    content_item["details"]["first_public_at"]
+  end
```

Comment:
just wondering if we should just use the specialist document presenter way of doing this everywhere so we don't hit that snag again until we have a reliable fix? question more than comment